arelle/plugin/validate/HMRC/ValidateHmrc.py

```diff
@@ -367,30 +434,120 @@ def validate(self) -> None:
                 else:
```

Comment:
The lines 428 and 429 are the same.

Reply:
Updated and updated testcase invalid-lp-sec477.xbrl to verify